dask_jobqueue/sge.py

```diff
@@ -0,0 +1,97 @@
+import logging
+import os
+import sys
+
+from .core import JobQueueCluster
+
+logger = logging.getLogger(__name__)
+
+dirname = os.path.dirname(sys.executable)
+
+
+class SGECluster(JobQueueCluster):
+    """ Launch Dask on a SGE cluster
+
+    Parameters
+    ----------
+    name : str
+        Name of worker jobs. Passed to `$SGE -N` option.
+    queue : str
+        Destination queue for each worker job. Passed to `#$ -q` option.
+    project : str
+        Accounting string associated with each worker job. Passed to
+        `#$ -A` option.
+    threads_per_worker : int
```

Comment:
For all the parameters inherited from JobQueueCluster, you should use the same mechanism as in PBSCluster or SLURMCluster, with the docstrings module: `%(JobQueueCluster.parameters)s`

Reply:
Done.

Reply:
I believe the sge.py file still misses some import and annotation, e.g. 
https://github.com/dask/dask-jobqueue/blob/2e406ae462b5c82fa679075e45bb2bfa59615610/dask_jobqueue/pbs.py#L5
and
https://github.com/dask/dask-jobqueue/blob/2e406ae462b5c82fa679075e45bb2bfa59615610/dask_jobqueue/pbs.py#L10

You also still have the description of interface kw in the docstring!

Reply:
Ah right I fixed that, I am not so familiar with the docrep magic!